hathor/cli/merged_mining.py

```diff
@@ -28,22 +33,38 @@ def execute(args: Namespace) -> None:
     from hathor.merged_mining.status_api import make_app as make_status_app
     from hathor.merged_mining.debug_api import make_app as make_debug_app
     from hathor.merged_mining.bitcoin_rpc import BitcoinRPC
+    from hathor.merged_mining.digibyte_rpc import DigibyteRPC
 
     loop = asyncio.get_event_loop()
 
-    bitcoin_rpc = BitcoinRPC(args.bitcoin_rpc)
+    kwargs: Dict[str, Any] = {}
+    has_parent_address = False
+    if args.bitcoin_rpc:
+        kwargs['bitcoin_rpc'] = bitcoin_rpc = BitcoinRPC(args.bitcoin_rpc)
+        kwargs['payback_address_bitcoin'] = args.bitcoin_address
+        has_parent_address = bool(args.bitcoin_address)
+    elif args.digibyte_rpc:
+        kwargs['digibyte_rpc'] = digibyte_rpc = DigibyteRPC(args.digibyte_rpc)
+        kwargs['payback_address_digibyte'] = args.digibyte_address
+        has_parent_address = bool(args.digibyte_address)
+    else:
+        raise ValueError('should have at least one parent chain')
     hathor_client = HathorClient(args.hathor_api)
     # TODO: validate addresses?
     merged_mining = MergedMiningCoordinator(
-        bitcoin_rpc=bitcoin_rpc,
         hathor_client=hathor_client,
         payback_address_hathor=args.hathor_address,
-        payback_address_bitcoin=args.bitcoin_address,
-        address_from_login=not (args.hathor_address and args.bitcoin_address),
+        address_from_login=not (args.hathor_address and has_parent_address),
         min_difficulty=args.min_diff,
+        constant_difficulty=args.const_diff,
+        **kwargs,
     )
-    logger.info('start Bitcoin RPC', url=args.bitcoin_rpc)
-    loop.run_until_complete(bitcoin_rpc.start())
+    if args.bitcoin_rpc:
+        logger.info('start Bitcoin RPC', url=args.bitcoin_rpc)
+        loop.run_until_complete(bitcoin_rpc.start())
+    elif args.digibyte_rpc:
```

Comment:
We have the same `if...else` before. Could we start the RPC services there or it needs to be after the coordinator?

Reply:
I thought there would be some problem, but it's actually fine. I'll do this refactor, thanks.

Reply:
Done.